code/continuousIntegration.adoc

```diff
@@ -4,6 +4,6 @@
 
 We use Jenkins for continuous integration.
 
-*Show http://ci.jbpm.org/[the Jenkins jobs].* These are mirrors of a Red Hat internal Jenkins jobs.
+*Show https://docs.jbpm.org/snapshot/brokenCILink.html[the public Jenkins jobs].* These are mirrors of a Red Hat internal Jenkins jobs.
```

Comment:
Would it be perhaps better to just say right here that there no public jobs available as of now? Now the sentence kind of suggests there are the jobs and when click the link, then you get the error message.